.github/workflows/test.yml

```diff
@@ -42,9 +42,9 @@ jobs:
 
     strategy:
       matrix:
-        os: ['bionic-3.8', 'focal-3.8']
-        python-version: ['3.8']
+        os: ['focal-3.10', 'jammy-3.10']
         tests-env: ['tests', 'tests_nds']
```

Comment:
**Suggestion:** Consider using a matrix strategy for the operating system and Python versions to reduce redundancy and make it easier to add or remove versions in the future. [enhancement]
```suggestion
    strategy:
      matrix:
        os: ['focal', 'jammy']
        python-version: ['3.10']
        tests-env: ['tests', 'tests_nds']
```